### DISCUSSION TITLE
渲染在线谷歌地图时出现方块

### DISCUSSION THREAD

**Discussion by jeffreylee0807** [F9y4ng/GreasyFork-Scripts#276] [问答专区(Question / Answer)] [answered]
渲染在线谷歌地图时出现方块
网址，www.google.com/maps
![Snipaste_2023-10-26_14-51-48](https://github.com/F9y4ng/GreasyFork-Scripts/assets/10457453/a9cb97ca-6b79-485a-ae99-df0baede999b)

**Comment by F9y4ng** [comment, marked as answer]
旧版升级至新版的用户注意：在遇到样式问题时，请先尝试使用重置按钮重置数据，再重新配置相关参数，以使用最新过滤规则。

**Reply by jeffreylee0807** [reply]
解决了，太给力了
